### PR TITLE
Update privacy.md

### PR DESCRIPTION
**Pull request by msbemba**
Changed the Navigation settings for PS

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3733

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/admin/productivity/privacy.md [hunk 1]
```diff
@@ -72,8 +72,8 @@ You can also opt out of the people experiences area of Productivity Score. If yo
 
 To opt put:
 
-1. In the admin center, go to the  **Settings**  >  **Org Settings**, and under  **Services**  tab, select  **Reports**.
+1. In the admin center, go to the  **Settings**  >  **Org Settings**, and **Productivity Score**
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/productivity/privacy.md]:
```suggestion
1. In the admin center, go to **Settings**  >  **Org Settings** > **Productivity Score**.
```

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@Sirkkuw Please merge this PR. Thanks

**Pull request merged**